packages/hooks/src/__tests__/useGetCfdAccounts.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useRealTotalAssetCurrency from '../useTotalAssetCurrency';
```

Comment:
@shayan-deriv This file seems wrong 🤔

Reply:
oh. forgot to delete this one. thanks :pray: